tests/josegonzalez/Dotenv/LoaderTest.php

```diff
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Comment:
Hmm not sure, maybe a month, but would rather just do baby steps first tbh - like just get this out, see if there's any issues and just work from there really